### PR TITLE
[docs] CSS style improvements

### PR DESCRIPTION
**Pull request by timhoffm**
As a suggestion:

- increase paragraph line-height to 1.5 to give text a bit more room to breathe. 1.5 is a common choice in web page typography; e.g. rtd-sphinx-theme, furo and GitHub use this (pydata sphinx theme goes even to 1.65).
- Don't shrink the text in admonitions. They are equally important and thus should have the same size as regular paragraphs. For reference, e.g. pydata-sphinx-theme and rtd-sphinx-theme do this.
- Adapt the border color for all warning/error like admonitions. Warnings had a fixed color that did not fit well to the new admonition style.

### CODE REVIEW

**Comment by chrisjsewell** [comment]
> Don't shrink the text in admonitions. They are equally important and thus should have the same size as regular paragraphs. For reference, e.g. pydata-sphinx-theme and rtd-sphinx-theme do this.

For counter-reference though 😅 : https://pradyunsg.me/furo/kitchen-sink/admonitions/, the text is smaller 

They are meant to be "asides" semantically, so I don't know if I agree that they are "equally important" (and I think in general furo is nicer looking and maybe better to follow)

If nothing else, the "feature" admonitions on the landing page should still be smaller.

The other changes I think are good 👍

**Review by chrisjsewell — CHANGES_REQUESTED** [review]
see comment

**Comment by timhoffm** [comment]
Re: Font size of admonitions:

I believe we're all not experts on typography. I therefore checked what "big sites" do, assuming they have thought about these things.

> [!NOTE]  
> GitHub does not reduce the font size in admontions.

The same holds for [Microsoft Azure docs](https://learn.microsoft.com/en-us/azure/devops/repos/git/use-ssh-keys-to-authenticate?view=azure-devops).

Based on that, I say same-size is likely better, but I don't have a strong opinion.

**Comment by timhoffm** [comment]
> If nothing else, the "feature" admonitions on the landing page should still be smaller.

Ok, kept them in the smaller font.

File: doc/_themes/sphinx13/static/sphinx13.css [hunk 1]
```diff
@@ -647,6 +654,7 @@ then for larger screens align them two per-row. */
     flex: 1 1 100%;
     margin: 0 !important;
     background-color: white !important;
+    font-size: 0.9em;
```
Comment (chrisjsewell) [inline doc/_themes/sphinx13/static/sphinx13.css:657]:
```suggestion
```

**Review by chrisjsewell — APPROVED** [review]

**Comment by chrisjsewell** [comment]
I removed the increase in admonition text size for now, since I really do feel it makes the pages with many admonitions less readable

**Pull request merged**